lib/acme/resources/challenges/base.rb

```diff
@@ -1,6 +1,6 @@
 class Acme::Resources::Challenges::Base
 
-  attr_reader :client, :status, :uri, :token, :error
+  attr_reader :client, :status, :uri, :token, :error, :iterations
```

Comment:
Well, something reintroduced it, https://github.com/saltsa/acme-client/commit/b465740a72758c88d23d12f534eb322d75ec79ee perhaps?